util/util.go

```diff
@@ -1,9 +1,37 @@
 package util
 
-import "os/exec"
+import (
+	"errors"
+	"net"
+	"os/exec"
+	"strings"
+)
 
 // IsCommandAvailable check command exits.
 func IsCommandAvailable(cmd string) bool {
 	_, err := exec.LookPath(cmd)
 	return err == nil
 }
+
+func GetClientIp() (string, error) {
+	addresses, err := net.InterfaceAddrs()
+
+	if err != nil {
+		return "", err
+	}
+
+	for _, address := range addresses {
+		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
+				return ipnet.IP.String(), nil
+			}
+
+		}
+	}
+
+	return "", errors.New("can not find the client ip addresses")
+}
+
+func CountWords(s string) int {
+	return len(strings.Fields(s))
+}
```

Comment:
Code review:

1. The import statement is repeated twice, which can be consolidated into a single import statement.
2. The IsCommandAvailable() function checks whether a given command is available in the system or not, which is a good practice to prevent errors during runtime.
3. The GetClientIp() function retrieves the IP address of the client, but it only returns the first IPv4 address it encounters. This may not be the intended behavior and could lead to unexpected results.
4. The CountWords() function counts the number of words in a given string, which is a useful utility function.

Improvement suggestions:
1. Consolidate the import statement into a single line.
2. Consider adding error handling when retrieving the IP address to handle edge cases where multiple IP addresses are present or when the IP address is not retrievable for some reason.
3. Consider adding unit tests for each function to ensure that they are working as intended.